packages/ol/src/index.spec.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
```

Comment:
Can we rename this file to `ol.spec.tsx` instead of `index.spec.tsx` to match other packages?

Reply:
Done!